tests/unit/mainwindow/test_prompt.py

```diff
@@ -94,6 +96,33 @@ def test_backspacing_path(self, qtbot, tmp_path, get_prompt):
         prompt.item_focus('next')
         assert prompt._lineedit.text() == str(testdir / 'bar')
 
+    @pytest.mark.parametrize("keys,expected",
+                            [([], ['bar', 'foo', 'bat'].sort()),
+                            (['F'], ['foo'].sort()),
+                            (['Backspace', 'A'], ['bar', 'bat'].sort())])
```

Comment:
No need for the `.sort()` here - just sort the values in the list manually, they're constant after all.